RLTest/redis_std.py

```diff
@@ -57,8 +58,9 @@ def __init__(self, redisBinaryPath, port=6379, modulePath=None, moduleArgs=None,
         self.clusterNodeTimeout = clusterNodeTimeout
         self.tlsPassphrase = tlsPassphrase
         self.enableDebugCommand = enableDebugCommand
-        self.terminateRetries = None
-        self.terminateRetrySecs = None
+        self.protocol = protocol
+        self.terminateRetries = terminateRetries
```

Comment:
Consider adding a class variable to define whether to report an error or to silently force termination when `terminateRetries` were exhausted and doing a force termination (`process.kill()`)